docs/v3/core/v3.0.rst

```diff
@@ -1160,13 +1099,18 @@ Optional keys:
 For example, the JSON document below defines an explicit group::
 
     {
-        "zarr_format": "https://purl.org/zarr/spec/v3#group",
+        "zarr_format": 3,
```

Comment:
I think we need something to explicitly distinguish between arrays and groups, rather than having to do it based on the presence of data_type or other array properties.  Note that I had changed `zarr_format` to be different for arrays and groups ( "https://purl.org/zarr/spec/v3#array" vs "https://purl.org/zarr/spec/v3#group").

Reply:
I'd rather put that into a different key than the version, if there are no objections. How about `node_type`, which is either `group` or `array`?